### DISCUSSION TITLE
请教各路大神，怎么部署像AI使用浏览器打开的界面化操作？

### DISCUSSION THREAD

**Discussion by PANLI77** [RapidAI/TableStructureRec#97] [Q&A]
请教各路大神，怎么部署像AI使用浏览器打开的界面化操作？

**Comment by Joker1212** [comment]
没懂，有图例吗？

**Reply by PANLI77** [reply]
也就是本地部署

**Reply by Joker1212** [reply]
源码下载后，下载模型，在初始化时传入模型path就行了，模型可以在这里找到https://github.com/RapidAI/TableStructureRec/releases/tag/v0.0.0
